src/utils/hooks/useFetchProjects.ts

```diff
@@ -15,19 +14,15 @@ const useFetchProjects = (
   const [projects, setProjects] = useState<CristinProject[]>([]);
   const [isLoading, setIsLoading] = useState(false);
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
+  const debouncedSearchTerm = useDebounce(searchTerm);
```

Comment:
jeg trodde egentlig hooks virka slik at argumentet bare ble brukt ved mount?
feks vil ikke `useState("init valu")` sette "init value" for hver render, men det er kanskje en `useState`-ting det eller? hvis ikke vil vel ikke denne verdien oppdateres

Reply:
nei, hvorfor skulle det kun bli brukt ved mount? det er jo en useEffect/funksjon i bunn. `searchTerm` blir oppdatert av `handleNewSearchTerm`, som igjen trigger kjøringen av `useDebounce`

Reply:
har bare tenkt det pga useState som bare tar inn startverdi of man må bruke en annen funksjon for å oppdatere.
hvis det ikke er sånn kan jeg forenkle litt på PRen for i18n av institutions også faktisk